src/Tools/CoreFx.Tools/GenerateDepsJson.netstandard.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.DotNet.Build.Tasks
+{ 
+    public partial class GenerateDepsJson: BuildTask
+    {
+        public bool IsNativeAssembly(string file, out AssemblyName result)
```

Comment:
Not in-box, but we have a portable implementation that works there. And I think we're already using it, because it's being restored to the net45 folder for me.